presidio-analyzer/Pipfile

```diff
@@ -5,7 +5,7 @@ name = "pypi"
 
 [packages]
 cython = "*"
-spacy = "==2.2.4"
+spacy = "==2.2.3"
```

Comment:
Why the downgrade?

Reply:
I upgraded it in my previous PR but didn't update it in other places so I realized there was a mismatch in versions. So I downgraded to what it was before.